custom/enikshay/management/commands/base_data_dump.py

```diff
@@ -118,9 +118,11 @@ def save_dump_to_blob(self, temp_path):
         return blob_dl_object.download_id
 
     def email_result(self, blob_ref):
+        if not self.recipient:
+            return
```

Comment:
You could probably just shortcut out of `handle` if `recipient` isn't provided, so it doesn't actually do the computation.